testdata/extension.config.yml

```diff
@@ -34,6 +51,18 @@ extensions:
         version: '^1.0.1'
       entries:
         main: 'src/index.tsx'
+  - uuid: 00000000-0000-0000-0000-000000000001
```

Comment:
Can you make the UUIDs unique. I don't think they are used internally to identify extensions, which I believe is done by list index, but it'd be good if they are unique in case we want to change that behaviour.